samples/csharp/getting-started/AnomalyDetection_CreditCardFraudDetection/Readme.md

```diff
@@ -0,0 +1,194 @@
+# Fraud detection in credit cards based on anomaly (or outlier) detection and existing PCA-transformed dataset
+
+| ML.NET version | API type    | Status     | App Type         | Data type | Scenario        | ML Task           | Algorithms     |
+|----------------|-------------|------------|------------------|-----------|-----------------|-------------------|----------------|
+| v1.1.0         | Dynamic API | Up-to-date | Two console apps | .csv file | Fraud Detection | Anomaly Detection | Randomized PCA |
```

Comment:
This would now be v1.2.0 I believe? 😄

Reply:
resolved in fcf77240b0acf610fe81cb01a68344d2d2f84d07